server/app/models/grid_service.rb

```diff
@@ -147,7 +147,7 @@ def stack_exposed?
   end
 
   def deploy_pending?
```

Comment:
Roger. I took the interpretation of `pending`too loosely. :)

How about `deploy_outstanding?`? At least in Cambridge dictionary it's defined as:
> Not yet solved or done